pkgs/development/tools/omnisharp-roslyn/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchurl
+, mono5
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "omnisharp-roslyn-${version}";
+  version = "1.32.8";
+
+  outputs = [ "out" ];
```

Comment:
This is likely redundant/implied when there are no other outputs listed.

Reply:
Indeed, it was unneeded.